drivers/volume/portworx/portworx.go

```diff
@@ -5016,9 +5016,11 @@ func addDrive(n node.Node, drivePath string, poolID int32, d *portworx) error {
 		driveAddFlag = fmt.Sprintf("-s %s", drivePath)
 		if poolID != -1 {
 			driveAddFlag = fmt.Sprintf("%s -p %d", driveAddFlag, poolID)
+		} else if poolID == -1 {
```

Comment:
just else is fine

Reply:
else means even if we uuid of poolid it will go to new pool so specifically used else if